core/dbt/adapters/base/impl.py

```diff
@@ -22,13 +22,20 @@
 import pytz
 
 from dbt.exceptions import (
```

Comment:
Is there a reason to do that other than not importing them individually?

Reply:
I think some Python codebases have policies around that but we don't. I guess if the import list gets really long it might be easier to just import the package. If we did import dbt..exceptions as exc we could specify every one as exc.SomeException.  I think some people prefer the specific imports because it makes it clearer which ones are used in this file. I'm kind of agnostic on the whole thing.

Reply:
> some people prefer the specific imports because it makes it clearer which ones are used in this file

I'm in this camp for sure.